qcengine/programs/dftd3.py

```diff
@@ -187,6 +191,22 @@ def parse_output(self, outfiles: Dict[str, str], input_model: "AtomicInput") ->
                 raise ResourceError("Cannot process ATM results from DFTD3 prior to v3.2.1.")
             elif re.match(r""" E6\(ABC\) /kcal,au:""", ln):
                 atm = Decimal(ln.split()[-1])
+            elif re.match(" analysis of pair-wise terms", ln):
+                D3pairs = np.zeros((full_nat, full_nat))
+                # Iterate over block
+                start = stdout.splitlines().index(ln) + 2
+                for l in stdout.splitlines()[start:]:
+                    data = l.replace("-", " -").split()
+                    # print(data)
+                    if len(data) == 0:
+                        break
+                    atom1 = int(data[0]) - 1
+                    atom2 = int(data[1]) - 1
+                    Edisp = Decimal(data[-1])
```

Comment:
Not sure we want to store this data as Decimal, these things are pretty explicitly floats.

Reply:
`Decimal` here is so as to not lose the info of how many digits of float we are getting back from the program.